Configuration/AlCa/python/GlobalTag.py

```diff
@@ -67,7 +68,8 @@ def GlobalTag(essource = None, globaltag = None, conditions = None):
             globaltag = globaltag[5:]
             if globaltag not in autoCond:
                 raise Exception('no correspondence for '+globaltag+'\navailable keys are\n'+','.join(autoCond.keys()))
-
+            if 'upgradePLS3' == globaltag:
+                sys.stderr.write('Warning: %s now points to %s, instead of POSTLS262_V1'%(globaltag,autoCond[globalTag]))
```

Comment:
@lgray there is a typo in here: 
`autoCond[globalTag]`  ==> `autoCond[globaltag]`